menu/menu.yaml

```diff
@@ -34,9 +34,9 @@
       Title: More tutorials
       Articles:
       - Url: tutorials/monitoring-setup
-        Title: Monitoring NServiceBus solutions
```

Comment:
I prefer the original. What's the issue with it?

Reply:
Decided to align the names with the demo and make it clear that this is about setting it up, not just information about it.